src/middleware/cookies.rs

```diff
@@ -7,6 +7,11 @@ use crate::{
     Context, Response,
 };
 
+/// `CookiesMiddleware middleware is required for `CookiesExt` implementation on `Context` to work.
```

Comment:
Could you add an empty line after this? That way we can have concise preview docs, but show more details when expanded.

Reply:
Done. Thanks.